### PR TITLE
WIP Add remark command

### PR DESCRIPTION
**Pull request by 500poundbear**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -96,6 +97,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case RedoCommand.COMMAND_ALIAS:
             return new RedoCommand();
 
+        case RemarkCommand.COMMAND_WORD:
```
Comment (donjar) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Add the ALIAS as well? To conform with other commands.

File: src/main/java/seedu/address/model/Model.java [hunk 2]
```diff
@@ -44,5 +44,4 @@ void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<ReadOnlyPerson> predicate);
-
```
Comment (donjar) [inline src/main/java/seedu/address/model/Model.java]: No reason to do this, right? Haha

**Review by donjar — APPROVED** [review]
LGTM! Just a few suggestions given

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 1 above)
Comment (500poundbear) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java]: yep next PR

**Pull request merged**